pennylane/pauli/pauli_arithmetic.py

```diff
@@ -257,19 +266,29 @@ def to_mat(self, wire_order, format="dense"):
         Rasies:
             ValueError: Can't get the matrix of an empty PauliSentence.
         """
+
+        def _pw_wires(w):
+            """To account for empty pauli_words which represent identity operations."""
```

Comment:
Needs a slightly more informative docstring.  And fixing the `inconsistent-return-statements`.

Reply:
Added 👍🏼 :